linalg/hypre.hpp

```diff
@@ -113,6 +121,9 @@ class HypreParVector : public Vector
    /// MPI communicator
    MPI_Comm GetComm() { return x->comm; }
 
+   /// Converts hypre's format to HypreParVector
+   void WrapHypreParVector(hypre_ParVector *y, bool owner=true);
```

Comment:
@pazner, before the `owner` parameter was added, the behavior was to not own the pointer. The new default may cause issues in code that already uses this branch. Maybe just switch the default to `false`?

Reply:
I think the default should be consistent between `WrapHypreParCSRMatrix` and `WrapHypreParVector`. I don't have a preference for whether it should be true or false by default, but it would be confusing if it was not the same for both. I don't think we need to worry too much about breaking backwards compatibility with un-merged code.

Reply:
Okay, being consistent between these two methods is good. Since the we already have the constructor `explicit HypreParMatrix(hypre_ParCSRMatrix *a, bool owner = true)`, let's keep the default to be `true` in both `Wrap*` methods.